base/Dockerfile

```diff
@@ -18,7 +19,7 @@ COPY finish-step.sh /
 RUN apk add --no-cache curl bash openjdk8-jre python3 py-pip nss libc6-compat \
       && ln -s /lib64/ld-linux-x86-64.so.2 /lib/ld-linux-x86-64.so.2 \
       && chmod +x *.sh \
-      && wget https://archive.apache.org/dist/spark/spark-${SPARK_VERSION}/spark-${SPARK_VERSION}-bin-hadoop${HADOOP_VERSION}.tgz \
```

Comment:
Yes, I know that the apache archive is a bit slower :( -- but I just integrated it within Github Actions and that is a process that is done once and then you/we do not need to rebuild it locally but rather just pull it from Docker Hub. The idea why we decided to go with Apache Archive as it keeps the archive indeed and not just the latest version. If at some point we want to rebuild  older version we can't do it with this mirror :( but with Apache Archive, we can always do so. I will prefer to keep it as it was. I know it is a bit slow, but we better keep it as is so that we can re-construct when needed. What do you think?

Reply:
Hi @GezimSejdiu, 

I think we can try to download file from faster repository and if it fails, we can point to archive repository. What do you think?

Reply:
That sounds like a great idea. Please allow this within Dockerfile and then I can move on with the merge.